userguide/vmware.rst

```diff
@@ -0,0 +1,131 @@
+.. index:: VMware Recommendations
+.. _VMware Recommendations:
+
+VMware Recommendations
+======================
+
+This section offers %brand% configuration recommendations and
+troubleshooting tips when using %brand% with a
+`VMware <https://www.vmware.com/>`__ hypervisor.
+
+
+.. _VMware guest:
+
+%brand% as a VMware Guest
+-----------------------------------
+
+This section has recommendations for configuring %brand% when it is
+installed as a Virtual Machine (VM) in VMware.
+
+#ifdef freenas
+To create a new %brand% Virtual Machine in VMware, see the
+:ref:`VMware ESXi` section of this guide.
+#endif freenas
+
+Be sure to install the VMware guest tools in new VMs. VMware provides
+instructions to
+`install VMware Tools <https://www.vmware.com/support/ws5/doc/new_guest_tools_ws.html>`__
+on different guest operating systems.
+
+Configure and use the
+`vmx(4) <https://www.freebsd.org/cgi/man.cgi?query=vmx>`__ drivers for
+VMs.
+
+Network connection errors for plugins or jails inside the %brand% VM can
+be caused by a misconfigured
+`virtual switch <https://pubs.vmware.com/vsphere-51/index.jsp?topic=%2Fcom.vmware.wssdk.pg.doc%2FPG_Networking.11.4.html>`__
+or
+`VMware port group <https://pubs.vmware.com/vsphere-4-esx-vcenter/index.jsp?topic=/com.vmware.vsphere.server_configclassic.doc_40/esx_server_config/networking/c_port_groups.html>`__.
+Make sure MAC spoofing and promiscuous mode are enabled on the switch
+first, and then the port group the VM is using.
+
+
+.. _Hosting Storage:
+
+Hosting VMware Storage with %brand%
+---------------------------------------------
+
+This section has recommendations for configuring %brand% when the system
+is being used as a VMware datastore.
+
+#ifdef truenas
+
+To improve :ref:`failover <Failover>` performance when there is an
+active iSCSI share, always configure :ref:`iSCSI Sharing <Block (iSCSI)>`
+with ALUA.
+
+#endif truenas
+
+Change the VM disk timeouts to prevent corruption of VM guest file
```

Comment:
s/Change the/Increase/
It's not to prevent corruption, it's to survive long disk operations or failover.